Artesian/_Services/QueryService.py

```diff
@@ -13,17 +13,20 @@
 from Artesian._Configuration.DefaultPartitionStrategy import DefaultPartitionStrategy
 
 class QueryService:
-    """ QueryService class contains query types to be created.
+    """ 
+        QueryService class contains query types to be created.
     
-        Returns:
-            Query Types."""
+        Attributes:
+            artesianConfig: The Artesian Configurazion.
```

Comment:
this class doesn't have this public Attribute.
The arg of the init is stored in a private __config thus not an Attribute.